3ArraysIndexIndex2Index3Average/3ArraysIndexIndex2Index3Average/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace _3ArraysIndexIndex2Index3Average
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Array fulfilled by indexes
+            int[] array1Degree = new int[10];
+            float sumArray1Degree = 0f;            
+            for (int index = 0; index < array1Degree.Length; index++)
+            {
+                array1Degree[index] = (int)Math.Pow(index, 1);
+                sumArray1Degree += array1Degree[index];
+            }
+            float averageArray1Degree = sumArray1Degree / array1Degree.Length;
+            Console.WriteLine("Array fulfilled by indexes: [{0}]", String.Join(", ", array1Degree));
```

Comment:
Plese, create function for output info about arrays

Reply:
Refactored by using methods.